simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/transactionctrlr/TxStopPointVariableAccessor.java

```diff
@@ -29,7 +29,7 @@
 
 public class TxStopPointVariableAccessor extends VariableAccessor {
 
-    public static final String NAME = "TxStopPoints";
+    public static final String NAME = "TxStopPoint";
```

Comment:
to chyba sie powtarza w TxStartPointVariableAccessor. Moze warto to dac gdzies do wsplnej zmiennej?